xija/gui_fit/plots.py

```diff
@@ -161,27 +161,17 @@ def clearLayout(layout):
 
 
 def annotate_limits(limits, ax, dir='h'):
+    from xija.limits import get_limit_color
```

Comment:
In general I think we should reserve local imports for dependencies that are optional to the code running or where import time is a priority for a heavy dependency like pandas. Same below with the local import of `re`, which is basically always imported already by the time you get to this level.

Reply:
Done.